README.md

```diff
@@ -1,44 +1,44 @@
 # NL Design Skeleton Application
 
-The NL Design Skeleton application is design for rapid application testing and prototype development on the NL Design System. It does so by providing a basic skeleton application with full NL Design system functionality that an developer can easily extend view locally and deploy to an online environment for demonstration purposes. the main benefits of this are
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend view locally and deploy to an online environment for demonstration purposes. The main benefit of this is:
 
-Development and (online) demonstration of prototypes without the need of an servers
-An ready out of the box basic  application that doesn't require configuration or setup and can be extended immediately.
+Development and (online) demonstration of prototypes without the need of a server.
+An out-of-the-box basic application that doesn't require configuration or setup and can be extended immediately.
 
-The application is available in two versions, a next.js version and an gatsby version. If you want to put your prototype online without setting up a server we recommend the gatsby version, since that can be deployed through gatsby pages.
+The application is available in two versions, a Next.js version, and a Gatsby version. We recommend the Gatsby version with deployment through Gatsby pages if you want to deploy your prototype without setting up a server.
 
 ## Getting started
 
-In order to set up your own project you will need an github account and be logged in, the simply click on “use this template” button. Tell github where you want to spin up your prototype and click “create repository from template”.
+To set up your own project, you will need a GitHub account and be logged in. Simply click on the "use this template" button. Tell GitHub where you want to spin up your prototype and click "create a repository from template".
 
+## Spinning up your local environment
 
-## Spinning up you  local environment
+You will need a git client, Node.js, and Yarn installed on your local machine for this part of the tutorial.
```

Comment:
Ik denk niet dat dit volledig klopt (correct me if I'm wrong):

(1) Je hebt geen Git Client nodig (ik lees dit alsof ik e.g. Git Kraken of SourceTree nodig zou hebben), je kan dit prima allemaal regelen via de command line;
(2) Volgens mij moet je `npm` óf `yarn` geïnstalleerd hebben op je machine, niet `Node.js` én `Yarn`.

Reply:
Edit: misschien dat je inderdaad `Node.js` én `npm` óf `yarn` nodig hebt.